monkey/agent_plugins/exploiters/wmi/src/wmi_remote_access_client.py

```diff
@@ -63,8 +63,8 @@ def login(self, credentials: Credentials, tags: Set[AgentEventTag]):
         tags.update(LOGIN_TAGS)
 
         try:
-            self._wmi_client.login(self._host, credentials)
             self._smb_client.login(credentials, tags)
+            self._wmi_client.login(self._host, credentials, self._options.smb_connect_timeout)
```

Comment:
Is switching this order intentional?

Reply:
Yep, I saw everywhere they were doing it like this. So I thought maybe it matters but nope. I will get it back to the original order